Python/thread_nt.h

```diff
@@ -399,9 +397,74 @@ PyThread_delete_key_value(int key)
     TlsSetValue(key, NULL);
 }
 
+
 /* reinitialization of TLS is not necessary after fork when using
  * the native TLS functions.  And forking isn't supported on Windows either.
  */
 void
 PyThread_ReInitTLS(void)
-{}
+{
+}
+
+
+/* Thread Specific Storage (TSS) API
+
+   Implementation part of platform-specific
```

Comment:
Rephrasing inline with the above suggestion: "Platform-specific components of TSS API implementation."

Reply:
Sure, I fix comments in one go.